### ISSUE TITLE
Connection Load Effects Result Table Labels

### ISSUE THREAD

**Issue by ozbuild** [idea-statica/ideastatica-public#12]
It looks like the Begin and End labels for the table displaying the load effects results is back to front. I think it should be the opposite to what is being displayed. ie the actual results being displayed are for the correct connected end, but the label says they are for the other end.

I have included a pdf showing the table and included the project and files
[Result Label Error.zip](https://github.com/idea-statica/ideastatica-public/files/6048117/Result.Label.Error.zip)

**Comment by vojtech-chalupa** [comment]
Hello, thank you for reporting this! I've looked into it and the labelling in the load table is indeed confusing. I will explain below, but we will make changes to make it clear in one of the following patches.

For a brief explanation - the (Begin / End) labels in the load table are not really related to the actual begin and end points of the beam. They were added to distinguish between the two ends of a continuous beam (C). Both "begin" and "end" loads on the beam C are related to the same one connection point, but the "begin" loads are for the side of the beam towards the begin point, and the "end" loads are for the side of the beam towards the "end" point. 
![image](https://user-images.githubusercontent.com/73105487/119103236-36ba6280-ba1b-11eb-97b7-a699b3135480.png)
Since when using the app as a standalone, without imports, the "Ended" beams always start in the connection point, the label in the load table always shows "end", as you always input the loads towards the end point of the beam. That's why the labels are "opposite".
